crates/extensions/c8y_http_proxy/src/messages.rs

```diff
@@ -6,8 +6,9 @@ use tedge_actors::ChannelError;
 use tedge_http_ext::HttpError;
 use tedge_utils::file::PermissionEntry;
 
-fan_in_message_type!(C8YRestRequest[C8yCreateEvent, C8yUpdateSoftwareListResponse, UploadLogBinary, UploadConfigFile, DownloadFile]: Debug, PartialEq, Eq);
-fan_in_message_type!(C8YRestResponse[EventId, Unit]: Debug);
+fan_in_message_type!(C8YRestRequest[GetJwtToken, IsUrlInCurrentDomain, C8yCreateEvent, C8yUpdateSoftwareListResponse, UploadLogBinary, UploadConfigFile, DownloadFile]: Debug, PartialEq, Eq);
+//HIPPO Rename EventId to String as there could be many other String responses as well and this macro doesn't allow another String variant
+fan_in_message_type!(C8YRestResponse[EventId, Unit, Bool]: Debug);
```

Comment:
> this macro doesn't allow another String variant

And cannot. If you receive pure `String`s from different sources how can you distinguish them to process them appropriately?

> `fan_in_message_type!(C8YRestResponse[EventId, Unit, Bool]: Debug);`

These response types are not really self contained. This is okayish for now as `c8y_http_proxy` is systematically used with a request-response pattern. But if for some reason we start to send concurrently several requests to the proxy, then we will have to replace these types with specific ones that give the context of the response.
An event id for what? What is true / false?

Reply:
> If you receive pure Strings from different sources how can you distinguish them to process them appropriately?

I was hoping to create a different alias like `type JwtToken = String` and use that aliased name as the enum variant. But I guess, both `EventId` and `JwtToken` aliases are substituted to `String` before the macro generates the variants, leading to the conflict. So I can either rename the existing `EventId` alias to just `String` so that it can be used everywhere or hand create the `C8YRestResponse` enum as follows:

```
enum C8YRestResponse {
    EventId(String),
    JwtToke(String),
    ...
```